app/services/helpline_service.rb

```diff
@@ -0,0 +1,39 @@
+require 'faraday'
```

Comment:
Great call on the service, thanks for fixing the services to emergency services!